src/task3/task3.js

```diff
@@ -0,0 +1,5 @@
+function isEven(n) {
```

Comment:
Exactly!
But there should be same pattern, everywhere:
```
function isEven(n) {
    if (n === 1) {
        return false;
    } if (n === 0) {
        return true;
    } else {
        return n > 0 ? isEven(n - 2) : isEven(n + 2);
    }
};
```

Reply:
ok, lesson learned, I suppose
but "Function writing pattern" was not immediately apparent to me, I'm afraid :(

Reply:
I am sorry I misguide you
Should be like this:
```
function isEven(n) {
    if (n === 1) {
        return false;
    } else if (n === 0) {
        return true;
    } else {
        return n > 0 ? isEven(n - 2) : isEven(n + 2);
    }
};
```